config/locales/idv/en.yml

```diff
@@ -21,6 +21,7 @@ en:
       ccn: Last 8 digits of a credit card
       city: City
       dob: Date of birth
+      dob_hint: 'example: 07/04/1999'
```

Comment:
as much as I like the July 4th nod, I wonder if it would aid UX to make the `dd` value something greater than 12, to visually enforce that it is the day, not the month.